SU-SWS/react_paragraphs
# V2 React Paragraphs

# READY FOR REVIEW

# Summary
- Upgraded a whole lot of things to make rows their own entity.
- This work can be merged in separately from the associated PRs. Since this is using a major release, it will require the upgrade in the respective PRs after the merge.

# Need Review By (Date)
- 8/31

# Urgency
- medium

# Steps to Test
1. do a site install
1. checkout the branch at https://github.com/SU-SWS/acsf-cardinalsites/pull/185
1. run database updates & config imports
1. verify the UI still works the same and all the content is the same.

# See Also
- https://github.com/SU-SWS/stanford_profile_helper/pull/18
- https://github.com/SU-SWS/stanford_profile/pull/275
